enhancer.js

```diff
@@ -0,0 +1,56 @@
+const { levels } = require("./level");
+// enhancer file with all the modules
+
+module.exports = {
+  success: item => {
+    const spell = item.enhancement + 1;
+    const ogName = item.name;
+
+    return { ...item, enhancement: spell, name: `[+${spell}] ${ogName}` };
+  },
+
+  fail: item => {
+    // The durability of an item cannot be less than 20 when the item's enhancement level is between +0 and +14.
```

Comment:
Great idea, grabbing the needed requirements from the README 👍 Makes it easier to capture all those conditions front and center.